hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metadata/HoodieBackedTableMetadataWriter.java

```diff
@@ -1125,7 +1130,10 @@ private void initialCommit(String createInstantTime, List<MetadataPartitionType>
       LOG.info("Committing " + partitions.size() + " partitions and " + totalDataFilesCount + " files to metadata");
     }
 
-    commit(createInstantTime, partitionToRecordsMap, false);
+    String commitInstantTime =
```

Comment:
should we be doing this in L1088 ? bcoz, this also gets into metadata records.

Reply:
Based on my understanding, `createInstantTime` is used in two places:
- For `files` partition: `getFilesPartitionRecords(createInstantTime, partitionInfoList, allPartitionRecord)`, `createInstantTime` is used to determine the file list till the instant time in data table.  So this should not have the suffix.
- For `bloom_filters` partition: `HoodieTableMetadataUtil.convertFilesToBloomFilterRecords(
            engineContext, Collections.emptyMap(), partitionToFilesMap, getRecordsGenerationParams(), createInstantTime)`, `createInstantTime` is stored in the metadata table record.  Not sure how important this is.

Reply:
This logic is moved to `HoodieBackedTableMetadataWriter.buildMetadataPartitions` now.